GETTING_STARTED.md

```diff
@@ -909,6 +909,9 @@ end
 Note that this approach works with `build`, `build_stubbed`, and `create`, but
 the associations will return `nil` when using `attributes_for`.
 
+Also, note that `instance` is not available within `initialize_with`, which may produce unexpected behaviour when 
```

Comment:
This isn't always true. `instance` would work fine if your custom `initialize_with` only instantiates the object in a custom way but doesn't assign all of the attributes.

How about:

```suggestion
Also, note that if you assign any attributes inside a custom `initialize_with` (e.g. `initialize_with { new(**attributes) }`), those attributes should not refer to `instance`, since it will be `nil`.
```